.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name: Main build
+
+on:
+    workflow_dispatch:
+    pull_request:
+        branches: [main]
+
+jobs:
+    build-gradle:
+        name: build and Test Gradle 
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'temurin'
+            
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+  

```

Comment:
I think we will encounter a 409 here as we'll be trying to publish to the same maven registry twice and overwriting whats already there. Can you please look into this link https://github.com/actions/delete-package-versions to clear up the old version before we try to push a new one?